drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/factory/ChromeDriverFactory.java

```diff
@@ -102,15 +102,30 @@ public Capabilities getCapabilities(WebDriverConfiguration configuration, boolea
 
         new ChromeDriverBinaryHandler(capabilities).checkAndSetBinary(performValidations);
 
-        ChromeOptions chromeOptions = new ChromeOptions();
-        CapabilitiesOptionsMapper.mapCapabilities(chromeOptions, capabilities, BROWSER_CAPABILITIES);
-
         // verify binary capabilities
         if (Validate.nonEmpty(binary)) {
```

Comment:
Shouldn't be this validation also part of setting ChromeOptions? Otherwise, (in the case of RemoveWebdriver) it may happen that you set non-executable binary...